starters/blog/gatsby-node.js

```diff
@@ -62,3 +62,11 @@ exports.onCreateNode = ({ node, actions, getNode }) => {
     })
   }
 }
+
+exports.onCreateWebpackConfig = ({ actions }) => {
```

Comment:
✂️ This change shouldn't be committed either.

Reply:
wanna push commit removing them?